source/postprocess/gravity_point_values.cc

```diff
@@ -0,0 +1,322 @@
+                               update_values   |
+                               update_gradients |
+                               update_quadrature_points |
+                               update_JxW_values);
+
+      // Get the value of the outer radius and inner radius
+      double model_outer_radius;
+      double model_inner_radius;
+      if (dynamic_cast<const GeometryModel::SphericalShell<dim>*> (&this->get_geometry_model()) != 0)
+        {
+          model_outer_radius = dynamic_cast<const GeometryModel::SphericalShell<dim>&>
+                               (this->get_geometry_model()).outer_radius();
+          model_inner_radius = dynamic_cast<const GeometryModel::SphericalShell<dim>&>
+                               (this->get_geometry_model()).inner_radius();
+        }
+      else if (dynamic_cast<const GeometryModel::Chunk<dim>*> (&this->get_geometry_model()) != 0)
+        {
+          model_outer_radius = dynamic_cast<const GeometryModel::Chunk<dim>&>
+                               (this->get_geometry_model()).outer_radius();
+          model_inner_radius = dynamic_cast<const GeometryModel::Chunk<dim>&>
+                               (this->get_geometry_model()).inner_radius();
+        }
+      else if (dynamic_cast<const GeometryModel::Sphere<dim>*> (&this->get_geometry_model()) != 0)
+        {
+          model_outer_radius = dynamic_cast<const GeometryModel::Sphere<dim>&>
+                               (this->get_geometry_model()).radius();
+          model_inner_radius = 0;
+        }
+      else
+        {
+          Assert (false, ExcMessage ("This initial condition can only be used if the geometry "
+                                     "is a sphere, a spherical shell, a chunk or an "
+                                     "ellipsoidal chunk."));
+          model_outer_radius = 1;
+          model_inner_radius = 0;
+        }
+
+      // Get the value of the universal gravitational constant
+      const double G = aspect::constants::big_g;
+
+      // now write all data to the file of choice. start with a pre-amble that
+      // explains the meaning of the various fields
+      const std::string filename = (this->get_output_directory() +
+                                    "output_gravity");
+      std::ofstream f (filename.c_str());
+      f << "# 1: position_satellite_r" << '\n'
+        << "# 2: position_satellite_phi" << '\n'
+        << "# 3: position_satellite_theta" << '\n'
+        << "# 4: position_satellite_x" << '\n'
+        << "# 5: position_satellite_y" << '\n'
+        << "# 6: position_satellite_z" << '\n'
+        << "# 7: gravity_x" << '\n'
+        << "# 8: gravity_y" << '\n'
+        << "# 9: gravity_z" << '\n'
+        << "# 10: gravity_norm" << '\n'
+        << "# 11: gravity_theory" << '\n'
+        << "# 12: gravity_diff (theoric-norm)" << '\n'
+        << "# 13: potential" << '\n'
+        << '\n';
+
+      // Temporary output during the build of this postprocessor WIP
```

Comment:
What means WIP? Did you include this only for debugging purposes? If so, please remove it. If it needs to stay in, it needs a better description of what you do here.